x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/helpers.tsx

```diff
@@ -5,12 +5,12 @@
  * 2.0.
  */
 
-import type { Prompt } from '../types';
+import { PromptResponse } from '@kbn/elastic-assistant-common/impl/schemas/prompts/bulk_crud_prompts_route.gen';
```

Comment:
```suggestion
import { PromptResponse } from '@kbn/elastic-assistant-common';
```
can these not be from the root?